mass_mailing_custom_unsubscribe/models/mail_unsubscription.py

```diff
@@ -0,0 +1,120 @@
+# Copyright 2016 Jairo Llopis <jairo.llopis@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, fields, models
+from odoo.addons.mass_mailing.models.mass_mailing import \
+    MASS_MAILING_BUSINESS_MODELS
+from .. import exceptions
+
+
+class MailUnsubscription(models.Model):
+    _name = "mail.unsubscription"
+    _description = "Mail unsubscription"
+    _inherit = "mail.thread"
+    _rec_name = "date"
+    _order = "date DESC"
+
+    date = fields.Datetime(
+        default=lambda self: self._default_date(),
+        required=True)
+    email = fields.Char(
+        required=True)
+    action = fields.Selection(
+        selection=[
+            ("subscription", "Subscription"),
+            ("unsubscription", "Unsubscription"),
+            ("blacklisting", "Blacklisting"),
+            ("de_blacklisting", "De-blacklisting"),
```

Comment:
haha, this is where I first saw that term https://github.com/odoo/odoo/blob/98f5ab8489cec60b82d73b5c65d51791bddcbca0/addons/mass_mailing/controllers/main.py#L139 and then I search a bit about that and i think that term is used in that scope. Anyway, I have changed it to avoid confusion